### ISSUE TITLE
Unable to Upgrade/Downgrade Google Android in-app subscriptions

### ISSUE THREAD

**Issue by superbre** [googlesamples/android-play-billing#25]
My problem is that every time I try to Upgrade/Downgrade a subscription calling the method launchPurchaseFlow setting the list of oldSkus, I always receive the error from Google: [DF-RPC-01].

When instead I try to purchase a brand new subscription from scratch (oldSkus list set null), everything works properly.

Anyone that has the same problem?

**Comment by scrobot** [comment]
Hi! Same problem. I found this code 
```
if (itemType.equals(ITEM_TYPE_SUBS)) {
            IabResult r = new IabResult(IABHELPER_SUBSCRIPTIONS_NOT_AVAILABLE,
                    "Subscriptions are not available.");
            flagEndAsync();
            if (listener != null) listener.onIabPurchaseFinished(r, null);
            return;
        }
```

in method 
```
launchPurchaseFlow(Activity act, String sku, String itemType, List<String> oldSkus,
                                   int requestCode, OnIabPurchaseFinishedListener listener, String extraData)
```

But why they are disabled?

**Comment by netomarin** [comment]
Hi @superbre 
This feature was added with version 5 of the in-app billing API (around February 2015).
But, is possible that your device haven't been updated.
To verify that the method is reported, send an isBillingSupported AIDL request.
Instead of using this method, we recommend that you use getBuyIntentExtraParams(), which provides additional functionality.

Hope I answered your question.
And because this issue is not exactly related to this sample, I'm closing the issue.
Next time you have technical question, feel free to make a question on Stackoverflow and use the tags google-play, in-app-purchase or in-app-billing.
Thanks.
